--- a/net/minecraft/game/entity/monster/EntitySkeleton.java
+++ b/net/minecraft/game/entity/monster/EntitySkeleton.java
@@ -3,10 +3,14 @@
 import net.minecraft.game.entity.Entity;
 import net.minecraft.game.entity.projectile.EntityArrow;
 import net.minecraft.game.item.Item;
-
+import net.minecraft.game.world.World;
 import util.MathHelper;
 
 public class EntitySkeleton extends EntityMob {
+	public EntitySkeleton(World world) {
+		super(world);
+	}
+
 	public final void updatePlayerActionState() {
 		super.updatePlayerActionState();
 	}
