--- a/net/minecraft/game/entity/monster/EntityZombie.java
+++ b/net/minecraft/game/entity/monster/EntityZombie.java
@@ -1,8 +1,13 @@
 package net.minecraft.game.entity.monster;
 
 import net.minecraft.game.item.Item;
+import net.minecraft.game.world.World;
 
 public class EntityZombie extends EntityMob {
+	public EntityZombie(World world) {
+		super(world);
+	}
+
 	public final void updatePlayerActionState() {
 		super.updatePlayerActionState();
 	}
