--- a/net/minecraft/game/entity/monster/EntityCreeper.java
+++ b/net/minecraft/game/entity/monster/EntityCreeper.java
@@ -2,8 +2,13 @@
 
 import net.minecraft.game.entity.Entity;
 import net.minecraft.game.item.Item;
+import net.minecraft.game.world.World;
 
 public class EntityCreeper extends EntityMob {
+	public EntityCreeper(World world) {
+		super(world);
+	}
+	
 	private int timeSinceIgnited;
 	private int lastActiveTime;
 	private int fuseDuration;
