--- a/net/minecraft/client/controller/PlayerControllerCreative.java
+++ b/net/minecraft/client/controller/PlayerControllerCreative.java
@@ -1,11 +1,16 @@
 package net.minecraft.client.controller;
 
+import net.minecraft.client.Minecraft;
 import net.minecraft.client.Session;
 import net.minecraft.game.entity.player.EntityPlayer;
 import net.minecraft.game.item.ItemStack;
 import net.minecraft.game.world.block.Block;
 
 public final class PlayerControllerCreative extends PlayerController {
+	public PlayerControllerCreative(Minecraft minecraft) {
+		super(minecraft);
+	}
+
 	public final void onRespawn(EntityPlayer entityPlayer1) {
 		for(int i2 = 0; i2 < 9; ++i2) {
 			if(entityPlayer1.inventory.mainInventory[i2] == null) {
