--- a/net/minecraft/game/entity/animal/EntitySheep.java
+++ b/net/minecraft/game/entity/animal/EntitySheep.java
@@ -3,9 +3,14 @@
 import net.minecraft.game.entity.Entity;
 import net.minecraft.game.entity.EntityLiving;
 import net.minecraft.game.entity.misc.EntityItem;
+import net.minecraft.game.world.World;
 import net.minecraft.game.world.block.Block;
 
 public class EntitySheep extends EntityAnimal {
+	public EntitySheep(World world) {
+		super(world);
+	}
+
 	public boolean sheared;
 
 	public final boolean attackEntityFrom(Entity entity1, int i2) {
