--- a/net/minecraft/client/render/EntityRenderer.java
+++ b/net/minecraft/client/render/EntityRenderer.java
@@ -369,11 +369,11 @@
 				double d78 = (double)(-MathHelper.sin(entityPlayerSP64.rotationYaw / 180.0F * (float)Math.PI) * MathHelper.cos(entityPlayerSP64.rotationPitch / 180.0F * (float)Math.PI)) * 4.0D;
 				double d39 = (double)(MathHelper.cos(entityPlayerSP64.rotationYaw / 180.0F * (float)Math.PI) * MathHelper.cos(entityPlayerSP64.rotationPitch / 180.0F * (float)Math.PI)) * 4.0D;
 				d41 = (double)(-MathHelper.sin(entityPlayerSP64.rotationPitch / 180.0F * (float)Math.PI)) * 4.0D;
-
-				for(f52 = (float)false; f52 < 8; ++f52) {
-					float f55 = (float)(((f52 & 1) << 1) - 1);
-					float f50 = (float)(((f52 >> 1 & 1) << 1) - 1);
-					f16 = (float)(((f52 >> 2 & 1) << 1) - 1);
+				
+				for(int i53 = 0; i53 < 8; ++i53) {
+					float f55 = (float)(((i53 & 1) << 1) - 1);
+					float f50 = (float)(((i53 >> 1 & 1) << 1) - 1);
+					f16 = (float)(((i53 >> 2 & 1) << 1) - 1);
 					f55 *= 0.1F;
 					f50 *= 0.1F;
 					f16 *= 0.1F;
